### DISCUSSION TITLE
Hubble tension

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#143] [Redshift Models]
Better measurements of the Cepheids agree with HST values so far.
[https://youtu.be/QHAaZQXZuLM?t=739](https://youtu.be/QHAaZQXZuLM?t=739)

**Comment by RedshiftDrift** [comment]
"An effective description of Laniakea and its backreaction: Impact on Cosmology and the local determination of the Hubble constant"

> We propose an effective model to describe the backreaction on cosmological observables induced by Laniakea, the gravitational supercluster hosting the Milky Way...  We find corrections on the distances of order ∼2−3%, resulting in a shift in the inferred best-fit values of the Hubble constant $H_0$ of order Δ $H_0^{SN Ia}$ ≈ +0.5 km/s/Mpc and Δ $H_0^{SBF}$ ≈ +1.1 km/s/Mpc, seemingly worsening the Hubble tension. 

[arXiv:2311.00215](https://arxiv.org/abs/2311.00215) [astro-ph.CO]

**Reply by bgreenman** [reply]
arXiv:2311.00215 shows that the Hubble constant and distant shift of the Laniakea supercluster is still needing constant adjustments

**Comment by RedshiftDrift** [comment]
"Circular reasoning: Solving the Hubble tension with a non-π value of π"
[https://arxiv.org/abs/2403.20219](https://arxiv.org/abs/2403.20219)

**Reply by mikehelland** [reply]
I'm sure you know this, but just in case anyone else doesn't, this is an April Fool's day prank.

**Reply by RedshiftDrift** [reply]
@mikehelland I know this is a joke, and it looks very much like other cosmology models I have seen on arXiv and elsewhere 😄

**Comment by RedshiftDrift** [comment]
"The Collapse of Cosmology's Sacred Cow, the Hubble Constant"
[https://www.youtube.com/watch?v=KZrAdBultJc](https://www.youtube.com/watch?v=KZrAdBultJc)

**Comment by RedshiftDrift** [comment]
"Measuring the Hubble constant through the galaxy pairwise peculiar velocity"
[arXiv:2412.04660](https://arxiv.org/abs/2412.04660)
"Our results yield $H_0 = 75.5 \pm 1.4~\mbox{km/s/Mpc}$"

That's a 5.4σ tension with the Planck result.

**Comment by RedshiftDrift** [comment]
"Calibrating the Tully-Fisher Relation to Measure the Hubble Constant"
[arXiv:2412.08449](https://arxiv.org/abs/2412.08449)

> We trace the SN Ia-based calibration used in the TF analysis to a study where $H_0$ was fixed to 70 km/s/Mpc rather than measured, [...] thus introducing a discrepancy with the other empirically calibrated indicators.
> Using the consistent Cepheid and TRGB calibration from B24 while removing the problematic data reduces the systematic error by a factor of two and results in $H_0$=76.3±2.1(stat)±1.5(sys) km/s/Mpc.
> We also show that most determinations of H0 measurements that replace Type Ia supernovae measurements with another far-field distance indicator yield H0>73 km/s/Mpc, reinforcing previous findings that the Hubble tension is not tied to any one distance indicator. 

Without the assumption $H_0 = 70$ km/s/Mpc, the value of the Hubble constant increases to 76 km/s/Mpc!

**Comment by RedshiftDrift** [comment]
*"Is our model of the universe wrong? – podcast"*
with Catherine Heymans, the astronomer royal for Scotland and a professor of astrophysics at the University of Edinburgh.
Nothing we don't already know, but explained to the less specialized public.
Skip forward to 1m30s https://www.theguardian.com/science/audio/2024/dec/17/is-our-model-of-the-universe-wrong-podcast

As for the question "Is our model of the universe wrong?" prof. Heymans does not give an answer.  But she's worried that her work on the dark sector might be wrong, and that is good for ACG.

**Comment by RedshiftDrift** [comment]
"Turning the Hubble tension into a crisis: New measurement confirms universe is expanding too fast for current models"
https://phys.org/news/2025-01-hubble-tension-crisis-universe-fast.html

Time to add another epicycle to LCDM!

**Reply by LifeIsStrange** [reply]
How about we add another epicycle to tired light models too?
The hubble tension is not specific to LCDM..
Not only does the tension exists independently of the CMB (BAO) but regardless, a difference in the history of the expansion speed of the universe (h0) is needed even in static unexpanding universes because in both cases you need to be able to tell the distance of things, via their redshift, in a coherent manner wherever you look and at whatever age of the universe.
The distance/redshift predictions of most TL models are different from LCDM but TL models have the same constraint, they must predict no difference in distance estimation between the CMB, BAO clusters and classical distance probes (supernovaes, etc)
Can you tell us the true distance of the Coma cluster you linked?
The fact the hubble tension is not discussed as a problem in alternative cosmologies is quite sad.
Also it is a misconception that there are necessarilly differences in distance estimation from static vs expanding universes, while that is most often true, some coasting expanding models, especially Rh=ct have the same prediction as TL, without making use of TL (unlike gupta hybrid model).

**Reply by mikehelland** [reply]
The Hubble tension in LCDM is making it fit "end-to-end", ie, from z=0 to the CMB at z~1100.

It's calculated to be sigma ~4 to 5. 

Which nonexpanding model are you claiming has this problem? And what sigma is calculated for the tension in this context?

**Reply by RedshiftDrift** [reply]
> a difference in the history of the expansion speed of the universe (h0) is needed even in static unexpanding universes - @LifeIsStrange 

There is no "history of the expansion speed of the universe" in a non-expanding universe, this comment is nonsense because a non-expanding universe does not expand!  Tired-light models don't have to explain any measurement interpreted as expansion because this is a false interpretation.  The Hubble tension is a tension between the 'distance-redshift relationship' *and* 'the Hubble constant derived from CMB fluctuations **interpreted in the expanding universe scenario**.  Clearly, in a non-expanding universe, the latter has no signification because there was no Big Bang.

The logarithmic nature of tired-light redshift $d \propto ln(1+z)$ accounts for the apparent "acceleration".  So tired-light models have no problems with the Hubble relation.  The Hubble tension is neither a problem nor exists in a non-expanding universe, that's why "the hubble tension is not discussed as a problem" because it is not.

As @mikehelland asked, which nonexpanding model are you claiming has a problem with the Hubble tension? when there is only one value of the Hubble constant that describes redshift in a non-expading model.

**Reply by HanDeBruijn** [reply]
@RedshiftDrift writes:
>there is only one value of the Hubble constant that describes redshift in a non-expanding model.

Really? I know at least two of these "Hubble constants" for Tired Light and they differ by a whole factor of two:

$$
H_{STz} = \frac{2}{\pi}\frac{\alpha h^2}{m_e^2 c} n_e \quad ; \quad H_{NTL} = \frac{1}{\pi}\frac{\alpha h^2}{m_e^2 c} n_e
$$

where $\alpha=$ fine structure constant, $h=$ Planck's constant, $m_e=$ electron mass, $c=$ speed of light, $n_e=$ electron density in the InterGalactic Medium.

In the [article](https://phys.org/news/2025-01-hubble-tension-crisis-universe-fast.html) the authors find a value of _76.5 kilometers per second per megaparsec_. Congratulations, Louis! (I've checked that) this is quite in concordance with your claimed [STz](https://github.com/orgs/a-cosmology-group/discussions/281) electron density $n_e \approx 0.3$. But with Lyndon Ashmore's [NTL](https://github.com/orgs/a-cosmology-group/discussions/277) theory we would have $n_e \approx 0.6$. And for [quite another reason](https://github.com/orgs/a-cosmology-group/discussions/281#discussioncomment-11854856) I would personally prever a value of $n_e\approx 0.5$. How many such (Hubble) tensions in a _non-expanding model_ shall be tolerated? @LifeIsStrange .

**Reply by mikehelland** [reply]
Han, you're comparing the Hubble constant of two different models.

That two different models produce two different values is not a tension.

The Hubble tension is that a single model, LCDM, produces two different values whose error bars don't overlap.

**Reply by HanDeBruijn** [reply]
>That two different models produce two different values is not a tension.

Mike, IMHO if two different models aim to describe one and the same phenomenon in Nature and give different values then that IS a tension.

**Reply by mikehelland** [reply]
Maybe in a colloquial sense. The scientific meaning of "tension" is specific here.

E.g., what do you calculate the sigma of the Hubble tension to be in STz?

You could compare its value from SNe measurements, to its value from CMB measurements. Except the latter doesn't exist.

**Reply by LifeIsStrange** [reply]
> a difference in the history of the expansion speed of the universe (h0) is needed even in static unexpanding universes

It is obvious that my sentence in a strict sense, is nonsensical, this is a tautological truism.
What I actually meant is that h0 is useful even in static universes, even though the _semantics_ of h0 becomes distinct from being a marker of the speed of expansion.

The Hubble Law is historical revisionism, if you read the biography of Edwin Hubble, you will see that he ended up believing the universe was static until his death. 
https://ui.adsabs.harvard.edu/abs/2009ASPC..413..255A/abstract

here is an example of the hubble constant in a static universe:
https://arxiv.org/abs/1005.0263

> which nonexpanding model are you claiming has a problem with the Hubble tension

Any model, for example New Tired Light from @tiredlyndon
let's look at his website:
http://tiredlight.org/about.htm
H0 is observed from the data he links, and can be measured both via redshift or via probes semi-independent of redshift (luminosity distance, angular distance, CMBtemperature evolution, parallax)
The data which is largely cosmologically independent gives a value here of 66 km/s
NTL can calculate H0 via the following basic equation with electron number density and some physical constants:
 H=(2nhr/m)
As more data to reduce electron number density uncertainty improves, his uncertainty reduces and ideally NTL must converge on the true value of h0, which here is supposedly 66km/s.
If as data improves NTL prediction diverges, NTL would be empirically falsified. This is whats makes it a great theory but also make it subject to wether the true value of H0 is 66 or 73km AKA the hubble tension.

As @HanDeBruijn has subsequently shown, other TL models can give different H0 values.

@mikehelland has also a point, that there is a distinction to makes, between **intra-tension** and **inter-models tension.**
Basically, a single model, such as LCDM, can gives multiple H0 values for different data, this self-contradiction, is what we call a concordance breakdown.
There has been a few papers about concordance breakdown and isn't only present for H0 but also for other parameters such as the S8 tension
https://arxiv.org/abs/2206.11447
self-contradiction is not necessarilly the end of a model, there are only to ways to save a model with concordance breakdown:
either the discrepancies/tension is explained via observational biases/systematics (dust attenuation, sample statistics, etc)
either one or multiple "constants" are actually not constant and evolves with redshift (or even with other parameters)

While inter models h0 tension are less dramatic than internal contradiction, they still constitute an important cosmological tension, meaning that if we agree on a correct value, we autoimatically falsify any model that predicted another value, hence the Hubble inter-models tension apply to static universes.

Moreover I do believe that concordance breakdown, internal hubble tension in static universes is very much possible and is underresearched. Concordance breakdown also has to do with the fact that LCDM is a more powerful, more expressive model than static cosmologies, sadly, because it predicts 6 cosmological parameters via monte-carlo permutations. The fact that nobody seemingly attempted predicting those cosmological parameters via static cosmologies, via monte-carlo is IMO one of the biggest failure of alternative cosmology and why there is such a failure (who actually attempted it? are people familiar with monte-carlo?) is a mystery.
Of course a model that predict more constants is more tested for breakdown...

Also crucially, if it is possible to derive H0 in a TL compatible way from various data, then intra-hubble tension is testable in static cosmologies and that is exactly the graph that TiredLyndon website shows.
But most cosmologists in TL seems to shield themselves from such falsifiability testing about their predictions of h0 on CMB, BAO and standard candles data.

This topic has much broader significance, it is not about wether the universe is expanding or wether the universe is static. It is about finding the first cosmological model that can answer the most basic and fundamental question: what is the true distance of the objects we look at?
And very few of you seems to understand that both LCDM and almost all alternative cosmologies have _failed_ and are empirically _falsified_.

The core topic of cosmology, the evaluation of distances is so fundamental and yet so overlooked..
Distance metrics are the following:
luminosity distance
angular distance
redshift
CMB remote temperature distance
parallax

The luminosity and angular distance both evolve distinctly in most (but not all) expanding vs static universes.
All distance metrics are subject to distinct systematics.
This allows the famous cosmological tests:
The Tolman test
The Distance Duality test
The Alcock-Paczynski test

Both the Tolman Test and Distance Duality test empirically falsify LCDM
Unless we assume extreme dust systematics coincidence, the only way to solves expansion are rh=ct or Gupta hybrid TL
https://arxiv.org/abs/2306.15680

However DESI DR1 Alcock-Paczynski empirically falsify both TL models and rh=ct, a fact that is never discussed here, despite the AP test being as conceptually solid as the Tolman Test

Therefore both cosmological paradigms are in their present forms, falsified, for pretty fundamental reasons about their light evolution.
This is considerably more significant than the Hubble tensions btw.

Cosmic time dilation contrary to popular myth is extremely noisy, the data does not allows to draw any reliable conclusion.
Homeostatic and past-eternal non cyclic universes are nearly ruled out via galaxy evolution.

Multiple anomalies (such as spin direction redshift dependence) would be worth discussing but that would be too long.

The redshift drift measurement is decades away, but the CMB temperature drift measurement is a real and underfunded possibility.

The main thing that allows progress is the recent observational measurements of the distant CMB temperature evolution (up to z=6)
A crucial insight about the Hubble tension is to see it as a temperature tension

Since TL is often modeled as d∝ln(1+z)
Two other metrics of distance evolves similarly with distance: cosmic time dilation and the CMB temperature
https://hal.science/hal-04368837/file/HaugTatumHubbleTensionCMBFeb16.pdf
section 3 page 5 shows there are two possible ways to model CMBtemp evolution with distance:
Tt = T0square(1 + z) OR Tt = T0(1 + z)

as in time dilation there is also the question of wether we must add 1-beta exponent, the value of beta being empirically close to zero, and the semantics of which, I do not understand.

Regardless, since CMB temp at z=6 is measured between 15 to 30 kelvins, both formulas are not yet ruled out and this means we currently have, in a model independent way, a distance uncertainty of high redshift objects, of a factor of 2. This means that objects could be up to twice closer than they appear.

Of note, this uncertainty bound is probably not a coincidence and seems to be exactly why current static cosmologies have the values they have, as Han said:
> I know at least two of these "Hubble constants" for Tired Light and they differ by a whole factor of **two**:

The true CMBtemperature evolution function will eventually be found (via better spectrographs) and this will resolve the hubble tension and should allows improved variants of the tolman test.
Do STZ and NTL even match Tt = T0square(1 + z) OR Tt = T0(1 + z) CMBT evolution? And if so why? Those measurements can no longer be ignored.

Another approach that is revolutionnary and that I have explained here
https://github.com/orgs/a-cosmology-group/discussions/277#discussioncomment-11882837
is that is is possible to calculate h0 purely from universal constants and the current CMB temp, this allows the most accurate determination of H0 ever and without systematics. 
It also allows to measure other constants in an independent manner, such as planck constant, the CMB temperature from H0, etc

**Reply by mikehelland** [reply]
>@mikehelland has also a point, that there is a distinction to makes, between intra-tension and inter-models tension.

There is no such thing as inter-models tension.

When scientists talk about a crisis, they don't mean it in the same sense as a humanitarian crisis, or mid-life crisis. They have a specific meaning in mind, from Kuhn's model of paradigm shift:
![image](https://github.com/user-attachments/assets/4f8dc6ed-5482-4dff-959c-016d6a6b77bd)

Like wise, "tension" isn't the same as one would use colloquially, as in a television drama. It has a specific meaning.

We can ignore all that, and use words the way we find convenient. But that kind of talk won't be taken seriously by anyone in the scientific community.

**Reply by budrap00** [reply]
The Hubble constant that isn't constant exists in the reality challenged expanding universe model of standard cosmology. In a non-expanding Cosmos H has neither relevance nor meaning.

**Reply by HanDeBruijn** [reply]
@mikehelland .
A reference is [Cosmology Slides: asking for reviews](https://github.com/orgs/a-cosmology-group/discussions/189).
You act as if &Lambda;CDM is **one** and the same coherent theory. Indeed that may appear so to an unprepared observer but it is **not**. In reality, the Big Bang framework is a hodgepodge of many different little theories. The trick is to assign each such a theory to a different **epoch**, epochs that are redundant anyway in a steady-state universe. It works like this: _Hey Mike, we all believe that you are not going to make trouble in our &Lambda;CDM community. We still have a few epochs for sale and we thought that the epoch with **Radiation alone** would be appropriate for you. In our ridiculous [Chronology of the universe](https://en.wikipedia.org/wiki/Chronology_of_the_universe) this means that the [Photon epoch](https://en.wikipedia.org/wiki/Photon_epoch) shall be assigned to your work. The epoch starts about 10 seconds from the Big Bang and ends 370,000 years later. What do you think?_

![kopij_2](https://github.com/a-cosmology-group/acg/assets/135433222/7f1db992-478a-4ce4-851c-9dbc7f483428)

**Reply by mikehelland** [reply]
>You act as if ΛCDM is one and the same coherent theory. Indeed that may appear so to an unprepared observer but it is not. In reality, the Big Bang framework is a hodgepodge of many different little theories. 

LCDM is an FLRW model, and it looks like this:

$$H = H_0 \sqrt{\Omega_r(1+z)^4 + \Omega_M(1+z)^3 + \Omega_k(1+z)^2 + \Omega_\Lambda}$$

If the universe only had matter, it would be simplified to this:

$$H = H_0 \sqrt{\Omega_M(1+z)^3}$$

If the univese only had radiation it would simplify to this:

$$H = H_0 \sqrt{\Omega_r(1+z)^4 }$$

etc

But the universe has all of those (well, except spatial curvature, k). We've entered, according to the theory, a dark-energy (lambda) dominated era. It was previously dominated by matter, and before that, radiation.

The equations in your image act as approximations for those different epochs. Not as individual theories.

**Reply by LifeIsStrange** [reply]
@RedshiftDrift As a followup to my previous extensive answer:
> there is only one value of the Hubble constant that describes redshift in a non-expading model.

How do you determine H0 in STz and what are its semantics ?
For example, in your excellent cosmology calculator:
https://cosmology.info/code/marmet_l/cosmology-calculators.html
You use the default parameter Hz which you call Hubble Redshift.
The source given is https://iopscience.iop.org/article/10.3847/1538-4357/ab1422
which is a 1% measurement of H0
Therefore in this static cosmology you use H0 = 74.03 ± 1.42 km s
But is Riess analysis not model dependent?
And even if it were not, the determination of H0 in STz from standard candles (here cepheids) does not preclude from the need to calculate H0 in a static universe via the CMB and via BAO, to determine wether there is a tension.
BAO H0 can be calculated for static universes, even the ones that do not model the CMB.
https://www.prophy.ai/article/184246062/

**Reply by RedshiftDrift** [reply]
> As a followup to my previous extensive answer - @LifeIsStrange 

I find no example of a Hubble tension within a tired light model in the [extensive answer](https://github.com/orgs/a-cosmology-group/discussions/143#discussioncomment-11884061).  As @mikehelland writes ["There is no such thing as inter-models tension."](https://github.com/orgs/a-cosmology-group/discussions/143#discussioncomment-11884184)  Indeed, different models are incommensurate (à la Kuhn).  No Hubble tension arises in a tired-light model from a comparison with a value of $H_0$ that is derived from the Big Bang model.

New Tired Light by @tiredlyndon predicts one value of $H_0$.  There can't be a tension since there's no other value to compare it to.

$\ $
> However DESI DR1 Alcock-Paczynski empirically falsify both TL models and rh=ct

That's incorrect. As stated in [this paper](https://iopscience.iop.org/article/10.1088/0004-637X/781/2/96) which uses the AP test: *"The static models are not totally excluded from the present analysis."*

$\ $
> the determination of H0 in STz from standard candles (here cepheids) does not preclude from the need to calculate H0 in a static universe via the CMB and via BAO, to determine wether there is a tension.

Incorrect, measurements of standard candles (cepheids and supernovae) are the only way to determine $H_z$ in STz.  Such a determination is incommensurate with the idea of using BAO, as there is no such thing as acoustic oscillations in the non-expanding paradigm. (As @tiredlyndon commented, "You are stuck in the Big Bang mindset with the Universe expanding." You must abandon that mindset that is contradictory to the 'no Big Bang paradigm'.)

$\ $
> How do you determine H0 in STz and what are its semantics ? - @LifeIsStrange 

In STz the photon energy obeys the differential equation $\mbox{d}E/E = -H_z \ \mbox{d}D$), where $H_z$ is the Hubble-Humason constant (units of $\mbox{Gpc}^{-1}$).  The idea was defined 88 years ago by [Nernst](https://doi.org/10.1007/BF01339902)!  Available in English [here](http://redshift.vif.com/JournalFiles/Pre2001/V02NO3PDF/V02N3NER.PDF).

From observations and the solution $D(z) = (1/H_z) \ \ln(1+z)$ to the differential equation given above, a fit gives the experimental value of $H_z$.  Measurements are done the same way as Hubble did them 100 years ago, which is what Riess is measuring today with cepheids+supernovae.

For the models we are concerned with, $\Lambda\mbox{CDM}$ and **STz**, the analyses yield numerical values that coincide, so it is appropriate to use the values already measured by astronomers since $H_0$ is close to $c \ H_z$. 

$\ $
> Therefore in this static cosmology you use H_0 = 74.03 ± 1.42 km/s/Mpc - @LifeIsStrange 

This is no longer the case, I haven't updated the cosmology calculator since 2023-4.  Now $H_z$ is ***calculated*** from the [cross section of an energy loss process](https://doi.org/10.48550/arXiv.2410.02036) (units of **km/s/Mpc** in the paper) and the [number density of electrons in the IGM](https://github.com/orgs/a-cosmology-group/discussions/281#discussioncomment-11908600).

$\ $
Not only there is no tension in any TL model, TL predicts the value of $\Lambda\mbox{CDM}$'s $\Lambda$ *without any adjustable parameter*!  Few people realize that the Hubble relation given by $\Lambda\mbox{CDM}$ (with its six++ parameters) is very close to the TL-predicted exponential function, so it is possible to derive the numerical value of $\Lambda\mbox{CDM}$'s $\Lambda$ by adjusting $\Omega_\Lambda$ as a parameter to fit the exponential function.

(I posted this image earlier but I can't find where... Here it is again below.)

The Hubble tension is the result of the incorrect assumption of expanding space.  There is no such problem with a TL-cosmology that uses a non-expanding model of space and predicts observations from fundamental physics.

![LCDM-fit-to-exp](https://github.com/user-attachments/assets/2caa2477-bd5d-4764-b6e5-6839c31d9827)

**Comment by LifeIsStrange** [comment]
Here is an alternative formalism to the S8 parameter that allows to define growth structure independently of the Hubble parameter hence uncoupling the S8 with the Hubble tension
https://arxiv.org/abs/2501.11571
